jerry-debugger/src/lib/cdt-proxy.ts

```diff
@@ -33,17 +33,21 @@ export const JERRY_DEBUGGER_VERSION = 'jerry-debugger/v0.0.1';
 export const DEVTOOLS_PROTOCOL_VERSION = '1.1';
 
 export class ChromeDevToolsProxyServer {
+  host: string;
+  port: number;
+  uuid: string;
+  jsfile: string;
```

Comment:
Add `readonly` to each? I think all get assigned in the constructor and you're not really supposed to re-assign later, correct?